packages/moleculer-bull/src/index.js

```diff
@@ -51,15 +50,94 @@ module.exports = function createService(url, queueOpts) {
 					this.$queues[name] = Queue(name, url, queueOpts);
 				}
 				return this.$queues[name];
+			},
+
+			/**
+			 * Set metrics for every queue
+			 * @param name
+			 */
+			addMetrics(name) {
+				// Collect metrics
+				try {
+					const queue = this.getQueue(name);
+					setInterval(async () => {
+						const {completed, active, delayed, failed, waiting} = await queue.getJobCounts();
+						this.broker.metrics.set("bull.completed", completed);
+						this.broker.metrics.set("bull.active", active);
+						this.broker.metrics.set("bull.delayed", delayed);
+						this.broker.metrics.set("bull.failed", failed);
+						this.broker.metrics.set("bull.waiting", waiting);
+					}, 1000);
+
+					// Collect duration job time
+					queue.on("completed", job => {
+						try {
+							if (!job.finishedOn) return;
+							const duration = job.finishedOn - job.processedOn;
+							this.broker.metrics.observe("bull.completed.duration", duration);
+						} catch (err) {
+							// silent
+						}
+					});
+				} catch(err) {
+					this.logger.warn("Unable to collect bull jobs metrics.", err);
+				}
 			}
 		},
 
 		created() {
 			this.$queues = {};
+			if(this.broker.metrics) {
+				this.logger.info("Metrics enabled");
+				this.broker.metrics.register({
+					type: "gauge",
+					name: "bull.completed",
+					labelNames: ["bull","queue"],
+					description: "Number of completed messages",
+					unit: "job"
+				});
+				this.broker.metrics.register({
+					type: "gauge",
+					name: "bull.active",
+					labelNames: ["bull","queue"],
+					description: "Number of active messages",
+					unit: "job"
+				});
+				this.broker.metrics.register({
+					type: "gauge",
+					name: "bull.delayed",
+					labelNames: ["bull","queue"],
+					description: "Number of delayed messages",
+					unit: "job"
+				});
+				this.broker.metrics.register({
+					type: "gauge",
+					name: "bull.failed",
+					labelNames: ["bull","queue"],
+					description: "Number of failed messages",
+					unit: "job"
+				});
+				this.broker.metrics.register({
+					type: "gauge",
+					name: "bull.waiting",
+					labelNames: ["bull","queue"],
+					description: "Number of waiting messages",
+					unit: "job"
+				});
+				this.broker.metrics.register({
+					type: "histogram",
+					name: "bull.completed.duration",
+					labelNames: ["bull","queue"],
+					description: "Time to complete jobs",
+					unit: "millisecond",
+					maxAgeSeconds: 600,
+					ageBuckets: 10
+				});
+			}
 		},
 
 		started() {
-			const setHandler = (handler, name)=>{
+			const setHandler = async (handler, name) => {
```

Comment:
Why need `async`? There is no `await` in the method.

Reply:
> Why need `async`? There is no `await` in the method.

already really no longer needed, changed the code and forgot to remove :(